suite/runsuite_common_post.cmake

```diff
@@ -1,5 +1,5 @@
 # **********************************************************
-# Copyright (c) 2011-2020 Google, Inc.    All rights reserved.
+# Copyright (c) 2011-2025 Google, Inc.    All rights reserved.
```

Comment:
nit: do we need update the copyright date of all the other files? Or do we usually do that with more substantial changes only?

Reply:
I went ahead and updated them (my emacs notifies me on saving generally but I edited these via cmdline so didn't notice).